ntoskrnl/kdbg/kdb_cli.c

```diff
@@ -301,6 +301,10 @@ ComponentTable[] =
     { "XSAVE",          DPFLTR_XSAVE_ID         },
     { "SE",             DPFLTR_SE_ID            },
     { "DRIVEEXTENDER",  DPFLTR_DRIVEEXTENDER_ID },
+//
+// ReactOS-specific components
+//
+    { "ROSUSBPORT",     DPFLTR_ROSUSBPORT_ID    },
```

Comment:
Why does this need a separate component?

Reply:
The current set of components has only `USBSTOR` for USB-related stuff. I can assign `usbport.sys` to some ID which does not have "USB" in it's name, but I think it may be confusing

Reply:
Same for `usbhub.sys`